src/code/z_actor.c

```diff
@@ -3230,30 +3232,30 @@ void Target_FindTargetableActorInCategory(PlayState* play, ActorContext* actorCt
 
     while (actor != NULL) {
         if ((actor->update != NULL) && ((Player*)actor != player) && CHECK_FLAG_ALL(actor->flags, ACTOR_FLAG_0)) {
-            // Enemy background music actor is updated here, despite not being too related to the Target system
+            // Enemy background music actor is updated here, despite not being too related to the Attention system
```

Comment:
Would this comment still be true? If we're interpreting this system as "Attention", then playing enemy music is a good audio cue to bring "attention" to the player that there's an enemy nearby

Reply:
fair point yeah. even more points for this system name :D

Reply:
Adding this to the list of responsibilities of the Attention System in z64actor.h
`- Playing enemy background music if a hostile actor is nearby`
Kind of softlaunching "hostile" as the term for `ACTOR_FLAG_2` here... soon tm